user_guide_src/source/general/managing_apps/002.php

```diff
@@ -1,4 +1,6 @@
 <?php
 
-$pathsConfig = 'app/Config/Paths.php';
+// Load our paths config file
+// This is the line that might need to be changed, depending on your folder structure.
```

Comment:
Sorry, missed this. But should this line be the same with line 6 below?

Reply:
https://github.com/codeigniter4/CodeIgniter4/blob/0f16cae244d8dd73f3505ec544055c5160362264/spark#L67-L70

https://github.com/codeigniter4/CodeIgniter4/blob/0f16cae244d8dd73f3505ec544055c5160362264/public/index.php#L30-L33

No, it's appropriate because it uses FCPATH.

Reply:
Sorry to be not clear. I mean this comment `// This is the line that might need to be changed, depending on your folder structure.` and the other one `// ^^^ Change this line if you move your application folder` both talk about changing the path, "sandwiching" the `require` line in between. Should only one be enough?

Reply:
I think only one comment is sufficient.

Reply:
PR sent #6389